userguide/storage.rst

```diff
@@ -2050,8 +2050,8 @@ can be replaced with a larger disk, waiting for the resilvering
 process to incorporate the new disk into the pool, then repeating with
 another disk until all of the original disks have been replaced.
 
-The safest way to perform this is to use a spare drive port or an
-eSATA port and a hard drive dock. The process follows these steps:
+The safest way to replace a drive is to use a spare drive port or an
```

Comment:
Please make a new ticket to rewrite this section: remove mention of eSATA (it's rare) and change "safest" to recommended. Add hot-swap trays and how they make shutting the system down unnecessary.

Reply:
Done: https://jira.ixsystems.com/browse/NAS-102865